app/components/DropNewPinButton.js

```diff
@@ -1,2 +1,33 @@
 
-//this should be a child of Map.js
+//this should be a child of ViewContainer
+
+import React, { Component, PropTypes, Text} from 'react-native';
```

Comment:
Let's remove PropTypes and Text from the import list since we aren't currently using them in this file.